expr/percentile_optimized.go

```diff
@@ -2,74 +2,40 @@ package expr
 
 import (
 	"fmt"
-	"time"
 
-	"github.com/getlantern/goexpr"
 	"github.com/getlantern/msgpack"
 )
 
 // PERCENTILEOPT returns an optimized PERCENTILE that wraps an existing
-// PERCENTILE and reuses its storage.
```

Comment:
We no longer attempt to reuse the existing percentile's storage, we assume separate storage for this percentile.